backend/api/Services/RobotService.cs

```diff
@@ -0,0 +1,32 @@
+using api.Context;
+using api.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace api.Services
+{
+    public class RobotService
+    {
+        private readonly FlotillaDbContext _context;
+
+        public RobotService(FlotillaDbContext context)
+        {
+            _context = context;
+        }
+        public async Task<Robot> Create(Robot robot)
+        {
+            await _context.Robots.AddAsync(robot);
+            await _context.SaveChangesAsync();
+            return robot;
+        }
+
+        public async Task<IEnumerable<Robot>> GetAll()
+        {
+            return await _context.Robots.ToListAsync();
+        }
+
+        public async Task<Robot?> GetRobot(string id)
+        {
+            return await _context.Robots.FirstOrDefaultAsync(robot => robot.Id.Equals(id));
+        }
```

Comment:
Maybe these should be renamed to "ReadAll" and "Read" in accordance with the CRUD naming conventions

Reply:
Done